bedrock/firefox/templates/firefox/interest-dashboard.html

```diff
@@ -24,9 +24,6 @@
   {{ fxfamilynav('desktop', '', 'dark') }}
 {% endblock %}
 
-{% block tabzilla_css %}{% endblock %}
-{% block tabzilla_js %}{% endblock %}
-
```

Comment:
This page also uses familynav not tabzilla. Changing it to page_css breaks the menu. Thinking a few PR's need to come out of the bug, one of them being replacing the familynav with tabzilla-static.